tests/lib/infusion/test-core/jqUnit/js/jqUnit-browser.js

```diff
@@ -28,6 +28,7 @@ var jqUnit = jqUnit || {};
         document.write("<scr" + "ipt src='" + injectPath + "?" + (new Date()).getTime() + "'></scr" + "ipt>"); /* from testswarm setup */ // jshint ignore:line
```

Comment:
Now we've got a proper module, we should really start sourcing Infusion from npm rather than checking it in in extenso